docs/quickstarts/excel-custom-functions-quickstart.md

```diff
@@ -0,0 +1,126 @@
+
+    - Choose a script type: `JavaScript`
+
+    - What do you want to name your add-in? `stock-ticker`
+
+    ![Yeoman generator for Office Add-ins prompts for custom functions](../images/12-10-fork-cf-pic.jpg)
+
+    The Yeoman generator will create the project files and install supporting Node components. The project files come from the [Excel-Custom-Functions](https://github.com/OfficeDev/Excel-Custom-Functions) GitHub repository.
+
+2. Go to the project folder.
+
+    ```
+    cd stock-ticker
+    ```
+
+3. Trust the self-signed certificate that is needed to run this project. For detailed instructions for either Windows or Mac, see [Adding Self Signed Certificates as Trusted Root Certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md).  
+
+4. Build the project.
+
+    ```
+    npm run build
+    ```
+
+5. Start the local web server, which runs in Node.js.
+
+    - If you'll be using Excel for Windows to test your custom functions, run the following command to start the local web server, launch Excel, and sideload the add-in:
+
+        ```
+         npm run start
+        ```
+        After running this command, your command prompt will show details about what has been done, another npm window will open showing the details of the build, and Excel will start with your add-in loaded. If you add-in does not load, check that you have completed step 3 properly.
+
+    - If you'll be using Excel Online to test your custom functions, run the following command to start the local web server:
+
+        ```
+        npm run start-web
+        ```
+
+         After running this command, another window will open showing you the details of the build. To use your functions, open a new workbook in Office Online.
+
+## Try out the prebuilt custom functions
+
+The custom functions project that you created by using the Yeoman generator contains some prebuilt custom functions, defined within the **src/customfunctions.js** file. The **manifest.xml** file in the root directory of the project specifies that all custom functions belong to the `CONTOSO` namespace.
+
+In your Excel workbook, try out the `ADD` custom function by completing the following steps in Excel:
+
+1. Within a cell, type `=CONTOSO`. Notice that the autocomplete menu shows the list of all functions in the `CONTOSO` namespace.
+
+2. Run the `CONTOSO.ADD` function, with numbers `10` and `200` as input parameters, by typing the value `=CONTOSO.ADD(10,200)` in the cell and pressing enter.
+
+The `ADD` custom function computes the sum of the two numbers that you specify as input parameters. Typing `=CONTOSO.ADD(10,200)` should produce the result **210** in the cell after you press enter.
+
+## Next steps
+
+Congratulations, you've successfully created a custom function in an Excel add-in! Next, learn more about the capabilities of custom functions and build a more complex add-in by following along with the Excel custom functions add-in tutorial.
+
```

Comment:
In the tutorial we should add a note that if you already did the quick start you can jump to the next section since it repeats a lot of steps here.

Reply:
I went ahead and made a link that jumps right to the right section for people - or should I change it back and have a note?